tf2onnx/graph.py

```diff
@@ -656,16 +670,32 @@ def reset_nodes(self, ops):
         self.contained_graphs = remained_sub_graphs
         self._nodes_by_name = {op.name: op for op in ops}
         self._output_to_node_name = {}
+        self._input_to_node_name = {}
         for op in ops:
             for op_output in op.output:
                 self._output_to_node_name[op_output] = op.name
+            if op.type == 'Placeholder':
+                inps = [op.name]
+            elif op.type == 'Const':
+                inps = [op.name]
```

Comment:
Why should Placeholder or Const ops be considered consumers of themselves?

Reply:
Removed placeholder.